charts/rstudio-connect/templates/rbac.yaml

```diff
@@ -1,7 +1,7 @@
 {{- if and (.Values.rbac.create) (.Values.launcher.enabled) }}
 {{ $namespace := $.Release.Namespace }}
 {{ $targetNamespace := default $.Release.Namespace .Values.launcher.namespace }}
-{{ $serviceAccountName := default .Values.rbac.serviceAccount.name (include "rstudio-connect.fullname" .) }}
+{{ $serviceAccountName := default (include "rstudio-connect.fullname" .) .Values.rbac.serviceAccount.name }}
```

Comment:
The changes of order here are just to ensure that if we have `.Values.rbac.serviceAccount.name` set, it will override the default of `rstudio-connect.fullname`?

Reply:
Yep, that's exactly right